packages/steps/src/LionSteps.js

```diff
@@ -56,8 +56,13 @@ export class LionSteps extends ObserverMixin(LionLitElement) {
   }
 
   get steps() {
-    let steps = this.children;
-    steps = Array.from(steps).filter(step => step.nodeName === 'LION-STEP');
+    const slots = this.shadowRoot.querySelectorAll('slot');
+    const slot = Array.from(slots).filter(element => element.name === '')[0];
```

Comment:
I believe you can do it with just one line `const slot = this.shadowRoot.querySelector('slot[name=""]')`

Reply:
made it simpler in the new implementation, no longer applicable